src/OpenTelemetry.Exporter.Console/OpenTelemetry.Exporter.Console.csproj

```diff
@@ -14,7 +14,7 @@
     <NoWarn>$(NoWarn),1591</NoWarn>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'net462'">
```

Comment:
Would it work to switch all of these `netstandard` \ `net462` checks to look at `.NETCoreApp` instead?

Example for this one but I think it would work in all spots on this PR:

```diff
-Condition="'$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'net462'"
+Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'"
```

Reply:
yes that is an option but my preference is to be explicit. Happy to change to fit your style if you would like.

Reply:
I'm fine either way. But if we do want to stick with the current style can we replace `net462` with `$(NetFrameworkMinimumSupportedVersion)` in the 2 not using the property so that part is consistent?

Reply:
@CodeBlanch change has been made to use the variable and keep it consistent. 👍